> -- Configuring x86-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:105 (message):
    Command failed: ninja -v
    Working Directory: D:/buildtrees/klein/x86-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      D:\buildtrees\klein\config-x86-windows-out.log

> Call Stack (most recent call first):
  D:/installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:351 (vcpkg_execute_required_process)
  ports/klein/portfile.cmake:9 (vcpkg_cmake_configure)
  scripts/ports.cmake:139 (include)
  
  It seems that the build system can't execute `ninja -v`? Should I add `DISABLE_PARALLEL_CONFIGURE` to `vcpkg_cmake_configure`?

--- a/ports/klein/find_simde_first.patch
+++ b/ports/klein/find_simde_first.patch
@@ -1,0 +1,36 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index c5d6f3f..a3ba161 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -29,16 +29,21 @@ endif()
+ option(KLEIN_BUILD_SYM "Enable compilation of symbolic Klein utility" ON)
+ option(KLEIN_BUILD_C_BINDINGS "Enable compilation of the Klein C bindings" ON)
+ 
+-include(FetchContent)
+-FetchContent_Declare(
+-    simde
+-    GIT_REPOSITORY https://github.com/simd-everywhere/simde
+-    GIT_TAG v0.7.2
+-    GIT_SHALLOW ON
+-)
+-FetchContent_GetProperties(simde)
+-if(NOT simde_POPULATED)
+-    FetchContent_Populate(simde)
++find_path(SIMDE_INCLUDE_DIR "simde/x86/sse4.1.h")
++if(NOT (${SIMDE_INCLUDE_DIR} STREQUAL SIMDE_INCLUDE_DIR-NOTFOUND))
++    set(simde_SOURCE_DIR ${SIMDE_INCLUDE_DIR}/simde)
++else()
++    include(FetchContent)
++    FetchContent_Declare(
++        simde
++        GIT_REPOSITORY https://github.com/simd-everywhere/simde
++        GIT_TAG v0.7.2
++        GIT_SHALLOW ON
++    )
++    FetchContent_GetProperties(simde)
++    if(NOT simde_POPULATED)
++        FetchContent_Populate(simde)
++    endif()
+ endif()
+ 
+ # The default platform and instruction set is x86 SSE3

--- a/ports/klein/portfile.cmake
+++ b/ports/klein/portfile.cmake
@@ -20,8 +20,8 @@ vcpkg_cmake_configure(
 vcpkg_cmake_install()
 vcpkg_fixup_pkgconfig()
 
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/klein RENAME copyright)
-file(INSTALL ${CMAKE_CURRENT_LIST_DIR}/usage DESTINATION ${CURRENT_PACKAGES_DIR}/share/klein)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL ${CMAKE_CURRENT_LIST_DIR}/usage DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT})
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
 

--- a/ports/klein/portfile.cmake
+++ b/ports/klein/portfile.cmake
@@ -4,6 +4,8 @@ vcpkg_from_github(
     REF 92023c6a946af84e716deb1488cad22048d3e08d
     SHA512 5d12ae143f07173992a6f9aa90539c4cb6c895a7169e5c086a10f78a31f7b2c9d64faf5ce1db014bd3badd92d24ff612dd2d2dc2f7508baac59307e3dccb6ebe
     HEAD_REF master
+    PATCHES
+        "find_simde_first.patch"
 )
 
 vcpkg_cmake_configure(

--- a/ports/klein/vcpkg.json
+++ b/ports/klein/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "klein",
-  "version-string": "2.3.0",
+  "version": "2.3.0",
   "description": "Klein is an implementation of P(R*_{3, 0, 1}), aka 3D Projective Geometric Algebra. It is designed for applications that demand high-throughput (animation libraries, kinematic solvers, etc).",
   "dependencies": [
     "simde",

--- a/versions/k-/klein.json
+++ b/versions/k-/klein.json
@@ -2,7 +2,7 @@
   "versions": [
     {
       "git-tree": "49c209248e2837e7eccefd0e6d94fe3f9e6beedd",
-      "version-string": "2.3.0",
+      "version": "2.3.0",
       "port-version": 0
     }
   ]

